keylime/config.py

```diff
@@ -16,6 +16,9 @@
 
 from keylime.common.version import str_to_version
 
+logging.basicConfig(level=logging.DEBUG)
+base_logger = logging.getLogger("keylime.config")
```

Comment:
One solution to avoid the circular dependency is to drop / refactor `keylime_logger` and setup the logger in the first import to `config`.  Maybe we need to create an issue to do this, as will clear a bit the codebase.

Reply:
Seems to be the best course of action. Please note that once this is implemented the changes to `config.py` , thanks to this PR, will be minimal (basically replace a few occurrences of `base_logger`)

Reply:
I would change the default logging level to INFO, because that is was I would expect when nothing is configured.

Yeah we can probably put the `keylime_logging` code into `config`.

Reply:
Do you want me to change `base_logger` to `INFO` meanwhile?

Reply:
Yes

Reply:
done